spec/index.html

```diff
@@ -957,6 +961,9 @@ <h6 class="defcap"><dfn>RDF vocabulary</dfn></h6>
           <li><code>rdf:subject</code></li>
           <li><code>rdf:predicate</code></li>
           <li><code>rdf:object</code></li>
+          <li><code><span style="background-color: Cyan;color: Black">rdf:TripleTerm</span></code></li>
```

Comment:
The domain of discourse of such a class is exactly _triple terms_ (as **syntactic** objects). E.g., in an unstar procedure, the generated "middle" bnode would be of type _triple term_, since it is exactly what it is meant to denote.

Reply:
It is possible that in some future WG, while defining the semantics for named graphs, the type Triple should be used for actual triples in the graph, not just triple terms.